src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -619,7 +625,7 @@ def create_table_and_insert(
             table_type="temporary",
         )
 
-        return [create_table, insert_into_statement(name, query)]
+        return [create_table, insert_into_statement(name, None, query)]
```

Comment:
Can we add name to these params, or at least comment what this `None` is?

Reply:
Updated to use explicit param names to call this function